plugins/org.python.pydev.core/src/org/python/pydev/core/docutils/PySelection.java

```diff
@@ -936,49 +939,57 @@ public static String getTextForCompletionInConsole(IDocument document, int docum
      *
      * @return the activation token and the qualifier.
      */
-    public static ActivationTokenAndQual getActivationTokenAndQual(IDocument doc, int documentOffset,
+    public static ActivationTokenAndQualifier getActivationTokenAndQualifier(IDocument doc, int documentOffset,
             boolean getFullQualifier, boolean handleForCalltips) {
         boolean changedForCalltip = false;
         boolean alreadyHasParams = false; //only useful if we're in a calltip
         int parOffset = -1;
         boolean isInMethodKeywordParam = false;
         int offsetForKeywordParam = -1;
-
         int foundCalltipOffset = -1;
         if (handleForCalltips) {
             int calltipOffset = documentOffset - 1;
-            //ok, in this case, we have to check if we're just after a ( or ,
             if (calltipOffset > 0 && calltipOffset < doc.getLength()) {
                 try {
-                    char c = doc.getChar(calltipOffset);
+                    int line = doc.getLineOfOffset(calltipOffset);
```

Comment:
Are this changes still needed?

Reply:
I don't think so. I reverted it in the last commit.